client/a8c-for-agencies/sections/marketplace/checkout/style.scss

```diff
@@ -0,0 +1,222 @@
+	margin-block-end: 64px;
+	font-size: 2.25rem;
+	line-height: 1.1;
+	font-weight: 600;
+}
+
+.checkout__main-list {
+	display: flex;
+	flex-direction: column;
+	gap: 32px;
+}
+
+.checkout__aside {
+	padding: 32px 0;
+
+	@include break-large {
+		max-width: 443px;
+		padding: 128px 48px;
+		background: var(--color-neutral-0);
+	}
+}
+
+.checkout__summary {
+	display: flex;
+	flex-direction: column;
+	gap: 16px;
+
+	hr {
+		width: 100%;
+		margin: 0;
+		padding: 0;
+		border: none;
+	}
+}
+
+.checkout__summary-pricing {
+	background-color: var(--studio-white);
+	border-radius: 8px; /* stylelint-disable-line scales/radii */
+	border: 1px solid var(--color-link);
+	padding: 1rem;
+
+	@media only screen and ( min-width: 782px ) {
+		margin-block-start: 32px;
+	}
+
+	@include break-xlarge {
+		margin-block-start: 24px;
+	}
+
+	.checkout__summary-pricing-discounted {
+		font-size: 1.25rem;
+		font-weight: 600;
+		color: var(--studio-gray-80);
+		line-height: 1;
+
+		@include break-xlarge {
+			font-size: 1.5rem;
+			font-weight: 700;
+			color: var(--studio-black);
+		}
+	}
+
+	.checkout__summary-pricing-original {
+		text-decoration: line-through;
+		font-weight: 400;
+		font-size: 1rem;
+		margin-inline-start: 8px;
+	}
+
+	.checkout__summary-pricing-interval {
+		font-size: 0.75rem;
+		font-weight: 400;
+		color: var(--studio-gray-60);
+		line-height: 2;
+	}
+}
+
+.checkout__summary-items {
+	margin: 0;
+}
+
+.checkout__summary-total {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	font-weight: 500;
+	font-size: 1rem;
+	line-height: 1.5;
+}
+
+
+.checkout__summary-notice {
+	color: var(--color-neutral-50);
+	font-size: rem(12px);
+	margin-block-end: 16px;
+
+	a {
+		text-decoration: underline;
+		color: var(--color-black);
+	}
+}
+
+.checkout__aside-actions {
+	margin-block-start: 32px;
+	display: flex;
+	flex-direction: column;
+	align-items: stretch;
+	gap: 16px;
+	text-align: center;
+
+	> .button {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		border-radius: 4px;
+		min-height: 40px;
+		font-weight: 600;
+	}
+
+	> .button.is-link {
+		color: var(--color-black);
+		text-decoration: underline;
+		font-weight: 400;
+	}
+}
+
+.product-info {
+	display: flex;
+	gap: 14px;
+
+	.product-info__icon {
+		display: none;
+		box-sizing: border-box;
+		width: 48px;
+		height: 48px;
+		background: linear-gradient(159.87deg, #f6f6f4 7.24%, #f7f4ea 64.73%, #ddedd5 116.53%);
+		border-radius: 4px;
+
+		@media only screen and ( min-width: 821px ) {
+			display: flex;
+			align-items: center;
+			justify-content: center;
+		}
+	}
+
+	.product-info__icon > img {
+		width: 22px;
+		height: auto;
+	}
+
+
+	.product-info__text-content {
+		width: 100%;
+	}
+
+	.product-info__header {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+	}
+
+	.product-info__label {
+		font-size: rem(16px);
+		font-weight: 700;
+	}
+
+	.product-info__count {
+		background-color: var(--studio-gray-0);
```

Comment:
[Non-blocker] Could we use color theme properties instead of this everywhere on this file?

Reply:
good catch 👍